### PR TITLE
SDK 15 on iOS

### PR DESCRIPTION
**Pull request by PeterPetrik**
following https://github.com/lutraconsulting/input-sdk/commit/300d1ae56c173daa492f52a8d2074fa8389c0a7d

fix https://github.com/lutraconsulting/input/issues/2001
fix https://github.com/lutraconsulting/input/issues/1983

also update of:
- exiv2=0.27.5
- expat=2.4.8 
- geodiff=1.0.6
- qgis=3.22.6

### CODE REVIEW

File: .github/workflows/ios.yml [hunk 1]
```diff
@@ -226,7 +226,11 @@ jobs:
             QMAKE_MAC_XCODE_SETTINGS+=qstyle \
             qstyle.name=CODE_SIGN_STYLE \
             qstyle.value=Manual
-
+            
+          # Use old deprecated build system in XCode 13
+          /usr/libexec/PlistBuddy -c "Add :DisableBuildSystemDeprecationDiagnostic bool" ./Input.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
+          /usr/libexec/PlistBuddy -c "Set :DisableBuildSystemDeprecationDiagnostic true" ./Input.xcodeproj/project.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings
+
```
Comment (tomasMizera) [inline .github/workflows/ios.yml:233]: Do we need to use the old build system?
Locally I applied this patch and it now generates the new build system https://codereview.qt-project.org/c/qt/qtbase/+/353119

Comment (PeterPetrik) [reply .github/workflows/ios.yml:233]: if it works without it now, I would keep the old one  for this commit. we should focus on qt6 path rather.

Comment (tomasMizera) [reply .github/workflows/ios.yml:233]: Ok, I agree, though it seems that it is neither fixed in qt6 https://bugreports.qt.io/browse/QTCREATORBUG-25368

**Review by tomasMizera — APPROVED** [review]
Good job!

**Pull request merged**